rock_paper_scissors/rock_paper_scissors.py

```diff
@@ -0,0 +1,36 @@
+""" Object-Oriented Rock Paper Scissors """
```

Comment:
IMO, this module would be better-placed in the root directory of the git repo. For one, it will reenable you to directly make use of the `__main__` check to start playing the game directly in your bash terminal instead of needing to first go into an interactive python shell.

Note that you'll need to rename it so it doesn't collide with the new `rock_paper_scissors` package. I suggest a name specific to the fact that it provides functionality for playing a CLI-based gamed (which will help disambiguate from web-based gameplay that we have yet to implement).

I wouldn't stress too much over the name, as we'll probably end up changing it fairly soon when we know more about the web implementation.